resources/js/alpine/tooltipComponent/utils/renderTooltip.ts

```diff
@@ -52,7 +52,7 @@ export function renderTooltip(
   );
 
   if (!options?.isBorderless) {
-    store.tooltipEl.classList.add('border', 'border-embed-highlight');
+    store.tooltipEl.classList.add('border', 'border-embed-highlight', 'bg-embed-highlight');
```

Comment:
Does this make the `border` and `border-embed-highlight` classnames redundant?
In other words, could they be removed?

Reply:
Did a test and yes they can be removed. I have noticed that the option `isBorderless` doesn't seem to do anything though. Setting it to true or false makes no difference.

Reply:
You will see it take effect on any card which has a loading indicator, such as game tooltips.

Reply:
I see the change now, my expectation was that it would effect all tooltips not just the loading spinner.